types/guild/ShoukakuPlayer.d.ts

```diff
@@ -33,7 +33,7 @@ export class ShoukakuPlayer extends EventEmitter {
   public setLowPass(values: { smoothing?: number } | null): ShoukakuPlayer;
   public setFilters(settings: ShoukakuFilter): ShoukakuPlayer;
   public clearFilters(): ShoukakuPlayer;
-  public resume(): ShoukakuPlayer;
+  public resume(options?: { noReplace?: boolean, pause?: boolean, startTime?: number, endTime?: number }): ShoukakuPlayer;
```

Comment:
Make this nullable

Reply:
The original options of playTrack didn't have the null in the typedef, I'll add it though if you want.

Reply:
don't pass null, it's only optional but should not be passed with null, that will throw error